.changeset/nasty-lemons-bow.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/auth-exp": patch
```

Comment:
Sorry if I wasn't clear, the whole file can be removed.

Reply:
Gone!